customers-service/src/test/resources/application.properties

```diff
@@ -1,2 +1,8 @@
 quarkus.http.test-port=9090
 eventuatelocal.kafka.bootstrap.servers=localhost:9092
+
+eventuateDatabase=h2
```

Comment:
Why was this change necessary?

Reply:
This test https://github.com/eventuate-examples/eventuate-quarkus-examples-customers-and-orders/blob/master/customers-service/src/test/java/net/chrisrichardson/eventstore/examples/customersandorders/customersservice/backend/CustomerPersistenceTest.java uses embedded database. Datasource should be configured. It transitively depends on eventuate-common-quarkus-in-memory-database where it is already defined, but properties are not applied, probably because they overwritten by properties in main/resources.

Reply:
However, current properties define not inmemory database. I will fix.

Reply:
fixed

Reply:
> Datasource should be configured. It transitively depends on eventuate-common-quarkus-in-memory-database

But why was this change needed now? The purpose of this PR is "Used eventuate platform dependencies"

Reply:
> However, current properties define not inmemory database. I will fix.

The same problem is here: https://github.com/eventuate-foundation/eventuate-common-quarkus/blob/master/eventuate-common-quarkus-in-memory-database/src/main/resources/application.properties#L5

Reply:
> but properties are not applied, probably because they overwritten by properties in main/resources.

I think it's important to

(a) understand what is going on - 'probably' is vague
(b) think about possible solutions since copy/pasting properties is not that great. For example, does Quarkus provide any mechanism for having multiple property files. For example, Spring has https://www.baeldung.com/spring-test-property-source

Reply:
> But why was this change needed now?

Without this change, build fails

>The same problem is here

I will fix

> understand what is going on - 'probably' is vague

I will investigate